aiodav/__init__.py,sha256=B8TUUMf6bes1yNuziJSHiuCUlVuXJ4q9B8lORHvK2ls,100
aiodav/client.py,sha256=tbjcNK7-CS7NYMyOdTZTzoSixMHLHWlo0XyGJlHIkb8,45720
aiodav/exceptions.py,sha256=S-OJHgNnY85kgl2Tegu-qgmQnSUY3EhbITZzzMS_M4E,2659
aiodav/urn.py,sha256=0W0Q346au-8QB49UT0hxmycsVTZeuylLCqQAOhVveCE,2113
aiodav/utils.py,sha256=z_Y5eZp2OxJmCXlO5xKtP5ZkWXBXEMGzq2gHFT6Sbcs,10551
aiodav-0.1.8.dist-info/LICENSE,sha256=m6FQVED-O2rDQzzTx89dp7PYYlfEd-aUGzfO7xqS9To,1084
aiodav-0.1.8.dist-info/METADATA,sha256=wAms8UhO-4ju6-YnQgXBA5SqR2qNZwmWsYLA8Me2Gvg,2441
aiodav-0.1.8.dist-info/WHEEL,sha256=ewwEueio1C2XeHTvT17n8dZUJgOvyCWCt0WVNLClP9o,92
aiodav-0.1.8.dist-info/top_level.txt,sha256=izb6vrcNoTNHhg5DB4BLgTD7d_m6lXBZo9dmeD4zjDQ,7
aiodav-0.1.8.dist-info/RECORD,,
